extensions/extensions-helm/aissemble-configuration-store-chart/templates/webhook-cleanup-job.yaml

```diff
@@ -0,0 +1,46 @@
+{{ if .Values.webhook.enable }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ index .Values "aissemble-quarkus-chart" "app" "name" }}-webhook-cleanup
+  annotations:
+    "helm.sh/hook": pre-delete, pre-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: {{ index .Values "aissemble-quarkus-chart" "app" "name" }}-webhook-cleanup
+    spec:
+      serviceAccountName: {{ .Values.webhook.serviceAccount.name }}
+      restartPolicy: OnFailure
+      containers:
+      - name: clean-webhook
+        image: {{ index .Values "aissemble-quarkus-chart" "deployment" "image" "repo" }}{{ index .Values "aissemble-quarkus-chart" "deployment" "image" "name" }}:{{ index .Values "aissemble-quarkus-chart" "deployment" "image" "tag" | default .Chart.AppVersion }}
```

Comment:
S: I'd rather use a slimmer image, especially if we're writing the shell script in the template and not executing something on the image. (Could argue that our current generation scripts might work better as a separate, targeted image.)

Reply:
Updated to use `curlimages/curl:latest`